src/OrchardCore/OrchardCore.DisplayManagement/TagHelpers/InputIsDisabledTagHelper.cs

```diff
@@ -0,0 +1,29 @@
+using Microsoft.AspNetCore.Mvc.TagHelpers;
```

Comment:
I assume this spilled in this PR

Reply:
I used this tag helper in the settings editor view to disable an input field based on a condition. Maybe this should be put somewhere else, or there is another solution?

Reply:
```
@{
  var disabled = Model.IsHttpsRequest ? "disabled" : (string) null;
}

<input disabled="@disabled"/>
```

Reply:
Your opinion on that?